setup.py

```diff
@@ -190,7 +186,9 @@ def run(self):
         coverage_cmd = 'python setup.py coverage %s' % self.get_args()
         doctest_cmd = 'python setup.py doctest'
         lint_cmd = 'python setup.py lint'
-        cmd = '%s && %s && %s' % (coverage_cmd, doctest_cmd, lint_cmd)
+        clean_cmd = 'python setup.py clean'
```

Comment:
I think there should not be a clean at the end of ci. If this is really necessary for tox, it can be inside tox.ini.